bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -47,16 +50,76 @@ public function __construct(
         GsOrderRepository $gsOrderRepository,
         KernelInterface $appKernel,
         MessageBusInterface $messageBus,
-        string $ownCloudFilesFolder
+        string $ownCloudFilesFolder,
+        TokenStorageInterface $tokenStorage
     ) {
         $this->entityManager = $entityManager;
-        $this->gsOrderRepository = $gsOrderRepository;
         $this->hammerService = $hammerService;
         $this->ownCloudService = $ownCloudService;
         $this->gsOrderRepository = $gsOrderRepository;
         $this->appKernel = $appKernel;
         $this->messageBus = $messageBus;
         $this->ownCloudFilesFolder = $ownCloudFilesFolder;
+        $this->tokenStorage = $tokenStorage;
+    }
+
+    /**
+     * Preset some values from user entity.
+     *
+     * @return \GraphicServiceOrder\Entity\GsOrder
+     */
+    public function prepareOrder()
+    {
+        $gsOrder = new GsOrder();
+        $token = $this->tokenStorage->getToken();
+        if (null !== $token) {
+            $user = $token->getUser();
+            $gsOrder->setFullName($user->getFullName());
```

Comment:
Consider chaining the setter calls: 
```
$gsOrder
  ->setFullName($user->getFullName())
  ->setAddress($user->getAddress())
  …
```
A convenience method, `setUser`, say, on `GsOrder` may also be useful for settings the user attributes on the order.

Reply:
Changed